src/app/api/timeline/bot/route.ts

```
@@ -0,0 +1,114 @@
+          data.hostUsername,
+          data.eventId
+        );
+        break;
+
+      case 'create_event_bots':
+        // Create event-specific bots for the event
+        await botSystem.createEventBot(data.eventId, BotType.EVENT_NOTIFY);
+        await botSystem.createEventBot(data.eventId, BotType.EVENT_HELPER);
+        
+        // Post welcome message to event timeline
+        const eventNotifyBot = await botSystem.getOrCreateBot(BotType.EVENT_NOTIFY, data.eventId);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)